addons/stock/models/stock_orderpoint.py

```diff
@@ -200,6 +200,11 @@ def _onchange_company_id(self):
                 ('company_id', '=', self.company_id.id)
             ], limit=1)
 
+    @api.onchange('route_id')
```

Comment:
If you take in account my comment about the optional `route_id` and look only on the `rules_ids`: it should be a onchange of the `rules_ids` ?

Reply:
Do you mean to add an additional `rule_ids` or to substitute it?

Reply:
It depends of the other comment, but substitute (because if it depends of `rule_ids`, it depends also of `route_id`, because rule_ids depends already of route_id)